roachtest: bump jepsen version

renatolabs opened the pull request:
This commit bumps the jepsen version now to include a fix that allows
jepsen to set custom range sizes. See:

https://github.com/cockroachdb/jepsen/pull/34

Epic: none

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/99669)
<!-- Reviewable:end -->

smg260 reviewed (approved):
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/99669)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @srosenberg)



<!-- Sent from Reviewable.io -->

renatolabs commented:
Sadly this will not work as Jepsen invokes `sudo` to invoke cockroach, which will erase any inherited environment we may set here.

renatolabs commented:
Jepsen test passes with these changes: <https://teamcity.cockroachdb.com/viewLog.html?buildId=9277206&buildTypeId=Cockroach_Nightlies_RoachtestNightlyGceBazel&tab=testsInfo&branch_Cockroach_Nightlies=99669>.

I'll wait for an approval in https://github.com/cockroachdb/jepsen/pull/34, merge that, generate a new uberjar, upload that to GCS, then update this PR to point to the new jar.

Commit:
roachtest: bump jepsen version

This commit bumps the jepsen version now to include a fix that allows
jepsen to set custom range sizes. See:

cockroachdb/jepsen#34

Epic: none

Release note: None

renatolabs commented:
Seems like the change in cockroachdb/jepsen#34 was not the full story. Running jepsen tests on this branch leads to ~50% of Jepsen tests still failing: <https://teamcity.cockroachdb.com/viewLog.html?buildId=9301827&tab=buildResultsDiv&buildTypeId=Cockroach_Nightlies_RoachtestNightlyGceBazel#testNameId2565973610725110455>.

This will need further investigation.

renatolabs commented:
All Jepsen tests pass now: <https://teamcity.cockroachdb.com/viewLog.html?buildId=9320590&buildTypeId=Cockroach_Nightlies_RoachtestNightlyGceBazel>.

@smg260 can I get a new stamp please?

smg260 reviewed (approved):
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/99669)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @srosenberg)



<!-- Sent from Reviewable.io -->

smg260 reviewed (approved):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/99669)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @srosenberg)



<!-- Sent from Reviewable.io -->

renatolabs commented:
bors r=smg260
